sources/packages/api/src/types/config.ts

```diff
@@ -1,10 +1,20 @@
 export interface IConfig {
   auth: IAuthConfig;
+  e2eTest: Ie2eTest;
 }
 
 export interface IAuthConfig {
   url: string;
   realm: string;
   clientId: string;
-  issuerUrl: string;
+  OpenIdConfigurationUrl: string;
+}
+
+export interface Ie2eTest {
```

Comment:
Can we move all test related config into test or testHelpers ?

Reply:
Done!